packages/network-controller/package.json

```diff
@@ -31,27 +31,27 @@
   "dependencies": {
     "@metamask/base-controller": "workspace:^",
     "@metamask/controller-utils": "workspace:^",
+    "@metamask/eth-json-rpc-infura": "^8.0.0",
+    "@metamask/eth-json-rpc-middleware": "^11.0.0",
+    "@metamask/eth-json-rpc-provider": "^1.0.0",
     "@metamask/swappable-obj-proxy": "^2.1.0",
     "@metamask/utils": "^5.0.2",
     "async-mutex": "^0.2.6",
     "babel-runtime": "^6.26.0",
-    "eth-json-rpc-infura": "^5.1.0",
```

Comment:
I looked at updating this in a separate PR, but it's probably safer not to, since this is the version web3-provider-engine expects and uses as a default.